tests/e2e/e2e_helm_install_test.go

```diff
@@ -0,0 +1,16 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test osm control plane installation with Helm", func() {
+	Context("Using default values", func() {
+		It("installs osm control plane successfully", func() {
+			// Install OSM with Helm
+			Expect(td.HelmInstallOSM()).To(Succeed())
```

Comment:
Can we also explicitly test if all of the osm components are running as expected?

- ensure osm controller pod status is in running state, this would catch any issue that cause the readiness checker to fail, e.g. cert issues, rbac issues, etc
- osm service exists
- crds exist, this would catch any change in our crds
- configmap exists with the default values, this would catch any default behavior changes in the chart
- mutatingwebhookconfig exists

Reply:
I added a schema file for values so that will help us catch config related issues around values like if a required value were missing, we'd get notified on the helm install test. Also does pattern matching on certificate-manager type, envoy log level and things like that.

Reply:
I can still add a test for the configmap default values though. Can't hurt.